src/accessors/IIterator.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Created by oliverjaegle on 22.06.17.
+ */
+
+export interface IIterator<T> {
+    next(): Array<T>;
```

Comment:
@graywolf336 I don't like Arrays as datatypes much, but in this case I feel them being appropriate